src/test/scala/com/nec/arrow/ArrowInterfacesTest.scala

```diff
@@ -41,11 +41,10 @@ final class ArrowInterfacesTest extends AnyFreeSpec {
       try {
         val vector = new Float8Vector("value", alloc)
         try {
-          val ndv = new non_null_double_vector
-          ndv.count = list.size
           val bp = new DoublePointer(list.toArray:_*)
-
-          ndv.data = bp.address()
+          val ndv = new non_null_double_vector()
+            .count(list.size)
```

Comment:
Worth to run the `Test` scope unit tests, they fail here for me.

Those tests should not rely on any external apps just JVM.

Reply:
Strange, I did run that scope, and it passed. Is there a reason CI didn't run on this PR?

Reply:
just triggered it

Reply:
JVM tests pass both on my local and in CI (in CI it failed later, due to a missing SonarCloud token)

Reply:
Unit test (JVM) seems to fail as per below.
Might recommend to move this test to `CMake` scope, should pass then I assume.

![image](https://user-images.githubusercontent.com/52247468/154574615-a253617c-bfbe-4eda-ad74-783c54d2b902.png)

Reply:
look at the logs. ArrowInterfacesTest passed (along with everything else). it only shows as failed because of SonarCloud error

Reply:
Thanks for pointing out. Makes sense. Any chance that you could commit to the SparkCyclone repo as a contributor? This would help the CI work fully and bring the PR into green by default.

It however fails on my local; screenshot here:

![image](https://user-images.githubusercontent.com/52247468/154583649-d925feae-7f31-4ee8-a0c7-9738b1526645.png)

Assume it's because the plug-in does not compile for Windows. Can you move the test to `CMake` scope? ie change file package to start with `com.nec.cmake`. The default `Test` scope is intended for pure-JVM (=platform-independent) things (after including the new plug-in it becomes platform-specific). The impact is that with pure-JVM aspects you always expect them to work without having to do any external/platform-specific things (eg `g++`).

Glad to see JavaCPP used here for this.

Reply:
Thanks, done.